lib/install.go

```diff
@@ -344,3 +344,18 @@ func canWrite(dir string) bool {
 	_ = os.Remove(fi.Name())
 	return true
 }
+
+// goPaths returns one or more Go paths.
+// If GOPATH is not set, $USER/go (the default GOPATH) is returned.
+func goPaths() []string {
+	path := os.Getenv("GOPATH")
+	if path == "" {
+		home := os.Getenv("HOME")
+		if home == "" {
+			panic("Cannot find either the GOPATH or HOME environment variables, please set at least one of them.")
```

Comment:
lets not panic, we can afford to return this as an error back up to the user.

Reply:
Agreed.... I am just going to return an `empty` slice.

I noticed this function was being used in `migrations` unnecessarily... so I removed its usage from there (it was necessary to find at least one path in that usage, but not in the install usage).
Notice that in migrations, the code already knew the path to the binary, it even tested it! No need to  look up the GOPATH variable in that case.